README.md

```diff
@@ -385,6 +456,118 @@ Installation Instructions
+    - **playbook_json.ces.yml**
+         -  Spectrum Scale Core, ZIMON, GUI, protocol (NFS, SMB), CallHome and Scale file audit logging. 
+ 
+ 
+- **set_json_variables.yml**
+
+    - The Playbook set variables from `set_json_variables.yml` that is read from `vars/scale_clusterdefinition.json`
+
+- **vars/scale_clusterdefinition.json**
+ 
+   - This file can be adjusted to your environment or created.
+   - Example scale_clusterdefinition.json` is separated into:
+   
+       - `scale_cluster`: 
+       - `node_details`: Variables that set's variables to each node (*host_vars*)
+       - `scale_storage`:
+       - `scale_callhome_params`:
+       - `scale_protocols`:
+       
+ 
+  ```json
+    {
+      "scale_cluster": {
+            "scale_cluster_name": "gpfs1.local",
+            "scale_cluster_profile_name": "gpfsprotocoldefaults"
+      },
+      "node_details": [
+        {
+          "fqdn" : host-vm1,
+          "is_protocol_node" :false,
+          "is_nsd_server" : false,
+          "is_quorum_node" : false,
+          "is_manager_node" : false,
+          "is_gui_server" : false,
+          "is_callhome_node" : false,
+          "scale_zimon_collector" : false
+        },
+        {
+          "fqdn" : host-vm2,
+          "is_protocol_node" : false,
+          "is_nsd_server" : false,
+          "is_quorum_node" : false,
+          "is_manager_node" : false,
+          "is_gui_server" : false,
+          "is_callhome_node" : false,
+          "scale_zimon_collector" : false
+        },
+        {
+          "fqdn" : host-vm3,
+          "is_protocol_node" : false,
+          "is_nsd_server" : false,
+          "is_quorum_node" : false,
+          "is_manager_node" : false,
+          "is_gui_server" : false,
+          "is_callhome_node" : false,
+          "scale_zimon_collector" : false
+        }
+      ],
+      "scale_storage":[
+        {
+          "filesystem": "cesSharedRoot",
+          "blockSize": 4M,
+          "defaultMetadataReplicas": 1,
+          "defaultDataReplicas": 1,
+          "automaticMountOption": true,
+          "defaultMountPoint": /gpfs/cesSharedRoot,
+          "disks": [
+           {
+            "device": "/dev/sdb",
+            "nsd": "nsd1",
+            "servers": "host-vm1",
+            "usage": dataAndMetadata,
+            "pool": system
+           }
+          ]
+        }
+      ],
+      "scale_callhome_params":{
+          "is_enabled": false,
+          "customer_name": abc,
+          "customer_email": abc@abc.com,
+          "customer_id": 12345,
+          "customer_country": IN,
+          "proxy_ip":,
+          "proxy_port":,
+          "proxy_user":,
+          "proxy_password":,
+          "proxy_location":,
+          "callhome_server": host-vm1,
+          "callhome_group1": [host-vm1,host-vm2,host-vm3],
+          "callhome_schedule": [daily,weekly]
+      },
+      "scale_protocols":{
+          "smb": false,
+          "nfs": false,
+          "interface": [eht0,eth1]
```

Comment:
@olemyk Thanks . I was thinking not to keep interface option in the general JSON template because interface option is only applicable for ipv6 CES enablement.  please let me know your opinion. thanks

Reply:
@rajan-mis  After my knowledge, the interface mode should also work for IPV4!?  
So is this current Ansible code limitation?

Reply:
Yes , we are only executing interface mode if ipv6 is there

Reply:
Understand, have removed it from the readme.